backend/backend/common/message_dtos.py

```diff
@@ -0,0 +1,30 @@
+from pydantic import BaseModel, validator
+
+
+VERSION = '1'
+
+
+class ClientContent(BaseModel):
+    sender: str
+    text: str
+
+
+class AnswerContent(BaseModel):
+    sender: str
+    time_laps: float
+    answer: str
+
+
+class UpdateScoreContent(BaseModel):
+    player_id: str
+    new_score: str
+
+
+class Message(BaseModel):
+    timestamp: str
```

Comment:
Interesting, why not datetime?

Reply:
becuase the type should specify the type of the output of the validator- and the validator's output is a string

Reply:
I don't understand, why do I care what is the return type of the validator, this should represent the type I'm going to work with, to convert this to another type simply use regular validator